testing/trino-tests/src/test/java/io/trino/connector/system/runtime/TestSystemConnector.java

```diff
@@ -75,7 +75,10 @@ protected QueryRunner createQueryRunner()
                 .setSchema("default")
                 .build();
 
-        DistributedQueryRunner queryRunner = DistributedQueryRunner.builder(defaultSession).build();
+        DistributedQueryRunner queryRunner = DistributedQueryRunner.builder(defaultSession)
+                .setSingleCoordinatorProperty("http-server.http.port", "8080")
+                .setSingleBackupCoordinatorProperty("http-server.http.port", "8081")
```

Comment:
I did not notice that before. We should not need a specific port assignement.
With that said it may not make sense any more to specify if backup coordinator should be used or not based on existence of backup coordinator properties.
Please leave the `setSingleBackupCoordinatorProperty` method but add another one to explicitly specify that we want to use backup coordinator: `enableBackupCoordinator()`

Reply:
You are right. I have changed these ports to random ones. @losipiuk

Reply:
Is it ok? @losipiuk